docs/_sass/_colors.scss

```diff
@@ -1,4 +1,4 @@
-$color-blue: #0185FF;
+$color-blue: #8DC8FF;
```

Comment:
I don't think this one should change, since our brand blue has not changed. We are just using a lighter shade of blue for links.

Reply:
Oh I see. The links on the help site use this variable for their non-hover color. 
Here they are side by side:
| #8DC8FF | #0185FF |
|:---:|:---:|
| ![Screen Shot 2023-01-03 at 2 55 03 PM](https://user-images.githubusercontent.com/49007721/210454953-c3f91c58-2f17-4dab-beb4-45f1e19f5062.png) |  ![Screen Shot 2023-01-03 at 2 55 27 PM](https://user-images.githubusercontent.com/49007721/210454974-5c92ea54-d3ba-455f-884e-f83f214afc0e.png) | 


You're saying we want the darker one, correct?